UNRELEASED.md

```diff
@@ -2,6 +2,12 @@
 
 > Place your changes below this line.
   
+- **Breaking:**
+  - `BpkPageTitle` was removed.
+  
+- **Added:**
+  - `BpkNavBar`
```

Comment:
Can we also add that support for navigation icons/actions was added?

Reply:
Will do.